tomviz/InterfaceBuilder.cxx

```diff
@@ -25,6 +25,9 @@
 #include <QSpinBox>
 #include <QWidget>
 
+// Has to be included before vtk_jsoncpp.h
+#include "vtkConfigure.h"
+
 #include "vtk_jsoncpp.h"
```

Comment:
`vtk_jsoncpp.h` needs `VTK_BUILD_SHARED_LIBS` to be set correctly in order to define the export macros for MSVC right.  Otherwise you don't get the `dllimport` on all the jsoncpp symbols.  This was the cause for the duplicate symbol errors.  @robertmaynard helped me track it down.

It is only a problem if `vtk_jsoncpp.h` is the first vtk header included (or in this case the only one).